src/pages/Environments.js

```diff
@@ -169,9 +173,14 @@ const Environments = () => {
   useOnMount(() => { loadData() })
   usePollingEffect(withErrorIgnoring(refreshData), { ms: 30000 })
 
+  const getCloudProvider = cloudEnvironment => isApp(cloudEnvironment) ? 'Kubernetes' : cloudEnvironment.runtimeConfig.cloudService === 'DATAPROC' ? 'Dataproc' : cloudEnvironment.runtimeConfig.cloudService
+  const getCloudEnvTool = cloudEnvironment => isApp(cloudEnvironment) ? _.capitalize(cloudEnvironment.appType) : _.capitalize(cloudEnvironment.labels.tool)
```

Comment:
Breaking these lines helps me read the logic more easily.
```suggestion
  const getCloudProvider = cloudEnvironment =>
    isApp(cloudEnvironment) ? 'Kubernetes'
    : cloudEnvironment.runtimeConfig.cloudService === 'DATAPROC' ? 'Dataproc'
    : cloudEnvironment.runtimeConfig.cloudService
  const getCloudEnvTool = cloudEnvironment =>
    isApp(cloudEnvironment) ? _.capitalize(cloudEnvironment.appType)
    : _.capitalize(cloudEnvironment.labels.tool)
```

Reply:
Hmm... I really hate nested ternary expressions, I split the first into Utils.cond as well.